include/groonga/table.h

```diff
@@ -312,6 +312,24 @@ grn_table_cursor_get_key_value(grn_ctx *ctx,
                                void **key,
                                uint32_t *key_size,
                                void **value);
+/**
+ * \brief Set `value` to the current record according to `flags`
+ *
+ * \param ctx The context object
+ * \param tc Target cursor
+ * \param value Value to set
+ * \param flags
```

Comment:
📝 `grn_obj_set_value` can use the following flags.
But `grn_table_cursor_set_value` can handle `GRN_OBJ_SET`, `GRN_OBJ_INCR`, or `GRN_OBJ_DECR`.

https://github.com/groonga/groonga/blob/7d5ae0a35dc437697ecf689a2d2ccf978d8f2b3f/doc/source/reference/api/grn_obj.rst?plain=1#L70-L91